admin/timescale/create_tables.sql

```diff
@@ -9,6 +9,14 @@ CREATE TABLE listen (
         data            JSONB                    NOT NULL
 );
 
+CREATE TABLE listen_new (
```

Comment:
I dislike the name of this table, it isn't very future proof, IMHO.
listen_v2? (or is it v3 already?)

Reply:
the intent is to do an atomic swap once the migration is complete to rename this to listen and the existing listen table to listen_old.